README.md

```diff
@@ -0,0 +1,92 @@
+
+
+<!-- PROJECT LOGO -->
+
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+
+
+
+
+
+### Built With
+this project is built using IBM watson machine learning 
+
+
+
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+
+* python 
+* BAM account 
+    - BAM_API_KEY
+    - BAM_URL
+
+### Installation
+
+_Below is an example of how you can instruct your audience on installing and setting up your app. This template doesn't rely on any external dependencies or services._
+
+1. Get a BAM API Key at [https://ibmmid.com](https://www.ibm.com/account/reg/us-en/signup?formid=urx-19776&target=https%3A%2F%2Flogin.ibm.com%2Foidc%2Fendpoint%2Fdefault%2Fauthorize%3FqsId%3D5b380573-b65b-469c-b3d3-e60c31c89011%26client_id%3DMyIBMDallasProdCI)
+2. Clone the repo
+   ```sh
+   git clone <project gti>
+   ```
+3. Install python packages
+   ```sh
+   pip install -r requirement.txt
+   ```
+4. Create a new file `.env` and Enter your BAM_API_KEY and BAM_URL  
+   ```sh
+       BAM_API_KEY=<your key>
+       BAM_URL=<url>
+   ```
+
+
+
+
+
+<!-- USAGE EXAMPLES -->
+## Usage
+
+in order to run the API service  
+   ```sh
+        uvicorn main:app --reload
+   ```
+
+
+
+<!-- ROADMAP -->
+## Roadmap
+
+- [ ] 
+
+
+See the [open issues](https://github.com/othneildrew/Best-README-Template/issues) for a full list of proposed features (and known issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- CONTRIBUTING -->
+## Contributing
+
+
+
+
+
+<!-- LICENSE -->
+## License
```

Comment:
@thoraxe @ilan-pinto Any thoughts around having [Apache-2.0](https://opensource.org/license/apache-2-0/) license to our project?

Reply:
I merged in the Apache 2.0 license and removed the GPL license.

Reply:
@thoraxe done